docs/ops/convolution/DeformableConvolution_1.md

```diff
@@ -12,7 +12,7 @@ Output is calculated using the following formula:
 
   \f[
 
-  y(p) = \sum_{k = 1}^{K}w_{k}x(p + p_{k} + {\Delta}p_{k})
+  y(p) = \displaystyle{\sum_{k = 1}^{K}}w_{k}x(p + p_{k} + {\Delta}p_{k})
```

Comment:
I'm not sure why, but currently in the old version, the formula is not rendered.

new:

![image](https://user-images.githubusercontent.com/39924818/128858020-353d44d5-49b9-4599-8526-d8e1dc449650.png)